web/modules/custom/joinup_notification/config/install/message.template.col_arc_del_no_request.yml

```diff
@@ -6,7 +6,7 @@ label: 'Collection archive/delete without request'
 description: 'Template id 13 - This will be sent in case the deletion/archival happened without request'
 text:
   1:
-    value: "<p>Your @entity:bundle \"@entity:title\" has been @transition:request_action:past.</p>\r\n\r\n<p>The reason for being @transition:request_action:past is: @transition:motivation.</p>\r\n"
+    value: "<p>Your @entity:bundle \"@entity:title\" has been @transition:request_action:past.</p>@transition:archive:extra:owner"
```

Comment:
This change is because in case of a delete, the message that was removed should be removed as well until we have the delete confirmation page.